modules/core/pom.xml

```diff
@@ -349,11 +350,6 @@
 
                                         <property name="props.file" value="../../modules/core/target/classes/ignite.properties" />
 
-                                        <replaceregexp file="${props.file}" byline="true">
```

Comment:
Why we removed it?

Reply:
This property is unused. Create ticket for that https://ggsystems.atlassian.net/browse/GG-25762